core/src/test/java/gov/nist/secauto/metaschema/core/metapath/cst/BuildCstVisitorTest.java

```diff
@@ -317,15 +317,15 @@ static Stream<Arguments> testNamedFunctionRefCall() {
   @ParameterizedTest
   @MethodSource("testNamedFunctionRefCall")
   void testNamedFunctionRefCall(
-    @NonNull String metapath,
-    @NonNull IItem expectedResult) {
-  StaticContext staticContext = StaticContext.builder().build();
-  DynamicContext dynamicContext = new DynamicContext(staticContext);
-
-  IItem result = IMetapathExpression.compile(metapath, staticContext)
-      .evaluateAs(null, IMetapathExpression.ResultType.ITEM, dynamicContext);
-  assertEquals(expectedResult, result);
-}
+      @NonNull String metapath,
+      @NonNull IItem expectedResult) {
+    StaticContext staticContext = StaticContext.builder().build();
+    DynamicContext dynamicContext = new DynamicContext(staticContext);
+
+    IItem result = IMetapathExpression.compile(metapath, staticContext)
+        .evaluateAs(null, IMetapathExpression.ResultType.ITEM, dynamicContext);
+    assertEquals(expectedResult, result);
+  }
```

Comment:
Sorry this occurred from running `mvn -PCI install` during the PR prep process locally. I can roll this back and commit elsewhere if you would prefer, @david-waltermire.